datascience/tables.py

```diff
@@ -1332,30 +1332,29 @@ def with_column(self, label, values):
 
         Args:
             ``label`` (str): The column label. If an existing label is used,
-                the existing column will be replaced in the new table.
+            the existing column will be replaced in the new table.
```

Comment:
Ok, issue is that errors of the throws, i.e tracebacks to the assertions errors, aren't exactly equivalent... (I don't print the entire error message, only a snip-it). It should be fixed except for those cases now.